docs/source/release/v6.6.0/reflectometry.rst

```diff
@@ -7,23 +7,30 @@ Reflectometry Changes
 
 New Features
 ------------
-- :ref:`ReflectometryISISLoadAndProcess <algm-ReflectometryISISLoadAndProcess>` can now take a polarization efficiencies workspace to make polarization corrections. This workspace can be set via the ``Polarization Efficiencies`` combo box on the ``Experiment Settings`` tab.
+- :ref:`ReflectometryISISLoadAndProcess <algm-ReflectometryISISLoadAndProcess>` can now take a polarization efficiencies workspace to make polarization corrections. This workspace can be set via the ``Polarization Efficiencies`` combo box on the :ref:`Experiment Settings tab <refl_exp_instrument_settings>`.
```

Comment:
There may already be a plan to update this, and it may not be relevant to do as part of this PR, but just to flag I noticed that the screenshot on the Experiment Settings docs that this links to doesn't include the new Polarization Efficiencies combo box.

Reply:
That's a good point. I'm going to include it in this PR as I'm updating the Preview tab's image in this one anyway